lib/matrix.test.js

```diff
@@ -33,3 +33,21 @@ test('product test',()=>{
     ]
   });
 });
+
+test('transpose test', () => {
+  let m = new Matrix(3, 3);
+  let n = new Matrix(3, 3);
```

Comment:
The `transpose()` function should return a new `Matrix` so there's no need to initialize here.